### PR TITLE
Update GraphicMods

### PR DESCRIPTION
**Pull request by linckandrea**
Adds global "Native Resolution DOF" and "DOF Removal"
Adds bloom and DOF definitions for Pandora's Tower
The Last  Story: Update HUD mod and move DOF efb target to the proper group

- Normal
SX3P01_2023-02-05_21-48-45

- Native res Bloom and DOF
SX3P01_2023-02-05_21-48-39

- Skip Bloom and DOF
SX3P01_2023-02-05_21-49-15

- TLS HUD

SLSP01_2023-02-05_11-59-41

### CODE REVIEW

File: Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+	"meta":
+	{
+		"title": "Bloom and HUD Texture Definitions",
+		"author": "iwubcode"
```
Comment (iwubcode) [inline Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:5]: I think that's very nice of you to mention me but you can take the credit for this one ;)

File: Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                   "type": "efb",
+                    "texture_filename": "efb1_n11_320x240_1"
+                }
+            ]
+        }
+	],
+    "features":
+    [
+        {
+			"group": "DOF",
```
Comment (iwubcode) [inline Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:39]: We should move this to a "Native Resolution DOF" all target like we have for bloom.  See https://github.com/dolphin-emu/dolphin/blob/master/Data/Sys/Load/GraphicMods/All%20Games%20Native%20Resolution%20Bloom/all.txt and https://github.com/dolphin-emu/dolphin/blob/master/Data/Sys/Load/GraphicMods/All%20Games%20Native%20Resolution%20Bloom/metadata.json

File: Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+	"meta":
+	{
+		"title": "Bloom and HUD Texture Definitions",
```
Comment (iwubcode) [inline Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:4]: "Bloom and DOF Texture Definitions"

File: Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json (hunk 2 above)
Comment (iwubcode) [reply Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:39]: Do you feel comfortable making that change?

Comment (iwubcode) [reply Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:39]: I'd also suggest adding a "DOF Removal" like we do for bloom:  https://github.com/dolphin-emu/dolphin/blob/master/Data/Sys/Load/GraphicMods/All%20Games%20Bloom%20Removal/metadata.json .

**Comment by linckandrea** [comment]
Thank you for the tips

how it is now?

also updated TLS HUD(i wasn't able to identify the remain texure in the upper left)
and move the DOF efb value to the right group

File: Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+				{
+					"type": "efb",
+					"texture_filename": "efb1_n10_320x240_4"
+				},
+				{
+					"type": "efb",
+					"texture_filename": "efb1_n11_320x240_1"
+				}
+			]
+		}
```
Comment (iwubcode) [inline Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:34]: Missing a `]` after this to close the "groups" section

Comment (linckandrea) [reply Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json:34]: done

**Comment by Sam-Belliveau** [comment]
@dolphin-emu-bot rebuild

**Comment by iwubcode** [comment]
@linckandrea - tested this out.  Overall is very good but I am questioning if Pandora's Tower has DOF.  Last Story does and it is very obvious there.  But what we call DOF in Pandora's Tower just seems like more bloom?

**Comment by linckandrea** [comment]
Doesn't DOF bloom out of focus things?

because if open the tower travel menu the background is bloomed (check the grass in the bottom left)

- NO mods
SX3P01_2023-02-07_11-58-09

- No DOF, Bloom untouched
SX3P01_2023-02-07_11-58-19

- Native DOF, Bloom untouched
SX3P01_2023-02-07_11-58-32

**Review by iwubcode — APPROVED** [review]
Forgot to approve this.  My original concern was that there's some overlap between the two terms, likely due to how the system  can't easily single out a particular effect.  However, there's not much that can be done about that right now.  So this LGTM.  Great work!

**Pull request merged**